### PR TITLE
2.0.0-alpha

### LINKED ISSUE #33: SDK no longer compiles (internal dependency automatically updated)
# Bug report

## Describe the bug

The current SDK release automatically picked up a newer version of it's internal client:

-  - SVRFClient (1.3.2):
+  - SVRFClient (1.4.0):

The problem is that this causes compilation issues for 1.0.3:

/Pods/SvrfSDK/SvrfSDK/Source/SvrfSDK.swift:137:73: Cannot convert value of type 'String?' to expected argument type 'MediaAPI.StereoscopicType_search?'
/Pods/SvrfSDK/SvrfSDK/Source/SvrfSDK.swift:178:33: Incorrect argument label in call (have 'type:stereoscopicType:category:size:nextPageCursor:completion:', expected 'type:stereoscopicType:category:size:minimumWidth:completion:')
### Desktop

- OS: 
- Browser: 
- Version: 

### Mobile

- Device: 
- OS: 
- Browser: 
- Version:

### PR DESCRIPTION
**Pull request by AndreiEvstratenko**
# Pull Request

## Description

- Update SDK with SVRFClient changes. Set specific versions for dependencies

### Motivation and Context

- closes #33
- closes https://github.com/Svrf/svrf-wave/issues/2092

### Types of changes

- [ ] Bug fix
- [ ] New feature
- [x] Maintenance

### CODE REVIEW

**Comment by brentchow** [comment]
As per my slack with @AndreiEvstratenko 

> Plan:
> 
> 1) make backward compatible
> 2) publish new version SDK
> 3) merge pagination PR
> 4) upload new version of the app in the testflight

**Comment by AndreiEvstratenko** [comment]
As per my slack with @brentchow. Create 2.0.0-alpha version without backward compatibility, cause it's unreal make it well.

File: SvrfSDK/Source/SvrfSDK.swift [hunk 1]
```diff
@@ -135,8 +135,8 @@ public class SvrfSDK: NSObject {
 
             MediaAPI.search(q: query,
                             type: options.type,
-                            stereoscopicType: options.stereoscopicType?.rawValue,
-                            category: options.category?.rawValue,
+                            stereoscopicType: options.stereoscopicType,
```
Comment (brentchow) [inline SvrfSDK/Source/SvrfSDK.swift]: no longer need to send the enum's `rawValue`?

File: SvrfSDK/Source/SvrfOptions.swift [hunk 2]
```diff
@@ -59,19 +49,22 @@ public struct SearchOptions {
  */
 public struct TrendingOptions {
     public init(type: [MediaType]? = nil,
-                stereoscopicType: StereoscopicType? = nil,
-                category: Category? = nil,
+                stereoscopicType: MediaAPI.StereoscopicType_getTrending? = nil,
```
Comment (brentchow) [inline SvrfSDK/Source/SvrfOptions.swift]: Were these not defined before in the SvrfClient?

Comment (AndreiEvstratenko) [reply SvrfSDK/Source/SvrfOptions.swift]: No, they weren't. I updated SDK with new enums of library

File: SvrfSDK/Source/SvrfSDK.swift (hunk 1 above)
Comment (AndreiEvstratenko) [reply SvrfSDK/Source/SvrfSDK.swift]: No, because earlier library expected string, but now enum

**Pull request merged**